main.go

```diff
@@ -328,7 +329,7 @@ func main() {
 				if viper.IsSet("gomaxprocs") && viper.GetInt("gomaxprocs") > 0 {
 					runtime.GOMAXPROCS(viper.GetInt("gomaxprocs"))
 				} else {
-					runtime.GOMAXPROCS(runtime.NumCPU())
+					_, _ = maxprocs.Set(maxprocs.Logger(log.Printf))
```

Comment:
Think we can remove Logger option from this call as it does not print anything in this case and we are logging gomaxprocs set on Info log level several line below anyway: https://github.com/centrifugal/centrifugo/blob/b4744fd4dc5d651640ff5cd16031c66be5415006/main.go#L342